### DISCUSSION TITLE
What is the plugin about

### DISCUSSION THREAD

**Discussion by formigadiablo** [linuxdeepin/developer-center#3853] [Q&A | 问答板块]
What does the plugin do?

**Comment by Decodetalkers** [comment]
which plugin? control center or dde-dock?

**Comment by formigadiablo** [comment]
Control center

**Reply by BLumia** [reply]
which plugin exactly?

In general, the control center provides a framework that allows 3rd-parties to implement their own "settings" UI and integrate it to the control center app.
